doc/source/whatsnew/v0.18.0.txt

```diff
@@ -312,6 +312,24 @@ New Behavior:
    s.index
    print(s.to_csv(path=None))
 
+Latex Represenation
+^^^^^^^^^^^^^^^^^^^
+
+``DataFrame`` has gained a ``_repr_latex_`` method in order to allow
+for conversion to latex in a ipython/jupyter notebook using nbconvert. (:issue:`11778`)
+
+Note that this must be activated by setting the option ``display.latex.repr`` to ``True`` (issue:`12182`)
+
+.. ipython:: python
+
+   df = pd.DataFrame({'a': [1, 2, 3]})
+   df._repr_latex_()
+   with pd.option_context('display.latex.repr', True):
+       print(df._repr_latex_())
```

Comment:
I am wondering a bit if this example is useful. As you would never use this kind of code like that, so maybe it is a possibly confusing